engine/src/rl/selfplay.cpp

```diff
@@ -337,29 +341,61 @@ TournamentResult SelfPlay::go_arena(MCTSAgent *mctsContender, size_t numberOfGam
     tournamentResult.playerA = mctsContender->get_name();
     tournamentResult.playerB = mctsAgent->get_name();
     Result gameResult;
-    for (size_t idx = 0; idx < numberOfGames; ++idx) {
-        if (idx % 2 == 0) {
-            gameResult = generate_arena_game(mctsContender, mctsAgent, variant, true, "");
+    #ifdef MODE_STRATEGO
```

Comment:
This change makes sense, also for chess variants, because we plan to support opening-suites in the future.

Reply:
No change required here.